vpr/src/base/atom_netlist.h

```diff
@@ -257,6 +261,10 @@ class AtomNetlist : public Netlist<AtomBlockId, AtomPortId, AtomPinId, AtomNetId
     vtr::vector_map<AtomBlockId, const t_model*> block_models_;   //Architecture model of each block
     vtr::vector_map<AtomBlockId, TruthTable> block_truth_tables_; //Truth tables of each block
 
+    //INPAD and OUTPUT models
```

Comment:
It would be good to comment these new member variables and why they exist:

Something like:
// Input IOs and output IOs always exist and have their own architecture models.
// While their models are already included in block_models_, we also store direct
// pointers to them to make checks of whether a block is an INPAD or OUTPAD fast,
// as such checks are common in some netlist operations (e.g. clean-up of an input netlist).

Reply:
Fixed